src/languages/en.ts

```diff
@@ -2035,8 +2036,25 @@ export default {
             disconnect: 'Disconnect',
             disconnectTitle: 'Disconnect integration',
             disconnectPrompt: 'Are you sure you want to disconnect this integration?',
-            importing: 'Importing customers',
             enterCredentials: 'Enter your credentials',
+            connections: {
+                syncStageName: ({stage}: PolicyConnectionSyncStageParams) => {
```

Comment:
```suggestion
                syncStageName: (stage: PolicyConnectionSyncStage) => {
```
Pass the stage directly without an object

(same for Spanish)